NetKAN/KIS trad fr

```diff
@@ -0,0 +1,6 @@
+{
+    "spec_version" : 1,
+    "identifier"   : "<KIS trad fr>",
+    "$kref"        : "#/ckan/spacedock/<1157>",
+    "license"      : "<open-source>"
```

Comment:
This is not a valid license string. Please check the possible strings close to the bottom of the schema file [here](https://github.com/KSP-CKAN/CKAN/blob/master/CKAN.schema)